config/hsc/processCcd.py

```diff
@@ -19,6 +19,7 @@
 config.calibrate.astrometry.matcher.maxRotationDeg=1.145916
 config.calibrate.astrometry.wcsFitter.numRejIter=3
 
+config.charImage.astrometry.wcsFitter.order = 3
 config.calibrate.astrometry.wcsFitter.order = 3
```

Comment:
Are there separate `charImage` and `calibrate` astrometry runs?  If so, we should make sure the settings for both are the same (maybe put them in a loop?).